src/Operations.js

```diff
@@ -767,9 +774,11 @@ export function zipWithFactory(keyIter, zipper, iters) {
         steps = iterators.map(i => i.next());
         isDone = steps.some(s => s.done);
       }
+
```

Comment:
Added white space for breathing room, can remove if it is inconsistent in style.